src/Command/AbstractReferencesCommand.php

```diff
@@ -50,19 +49,8 @@ protected function getFiles(): iterable
                 throw new RuntimeException(sprintf('File "%s" does not seem to be a valid PHP class.', $file->getPathname()));
             }
 
-            foreach ($tagsToIgnore as $tagToIgnore) {
-                if ($reflectionClass->hasTag($tagToIgnore)) {
-                    continue 2;
-                }
-            }
-
-            // class is not an interface nor a trait, and has no protected/public methods nor properties
-            if (
-                !$reflectionClass->isTrait()
-                && !$reflectionClass->isInterface()
-                && !\count($reflectionClass->getMethods())
-                && !\count($reflectionClass->getProperties())
-            ) {
+            // class has tags to ignore or should be excluded (@see ClassParser::isExcluded)
+            if ($this->configuration->isExcluded($reflectionClass) || $reflectionClass->isExcluded()) {
```

Comment:
nitpick: it feels a bit weird that this is handled by the `ConfigurationHandler` and not by the `ClassParser` itself (by injection the configuration)